src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -43,9 +46,12 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
   const isCephDriver = formik.values.driver === cephDriver;
   const isCephFSDriver = formik.values.driver === cephFSDriver;
   const isDirDriver = formik.values.driver === dirDriver;
+  const isLvmDriver = formik.values.driver === lvmDriver;
   const isPowerFlexDriver = formik.values.driver === powerFlex;
   const isPureDriver = formik.values.driver === pureStorage;
+  const isZfsDriver = formik.values.driver == zfsDriver;
   const storageDriverOptions = getStorageDriverOptions(settings);
+  const canClusterSource = !isDirDriver && !isZfsDriver && !isLvmDriver;
```

Comment:
```suggestion
  const hasClusterWideSource = isCephDriver || isCephFSDriver;
```
I think defining this positively without negation is much easier to read.